### PR TITLE
DCAC-76: Add basic signPdf endpoint

### PR DESCRIPTION
**Pull request by VaughanJackson123**
* For now, the endpoint attempts to access the unsigned document at the location specified, and if it finds it, returns the location of the unsigned document, rather than the location of the signed document as it will later.
* Another temporary measure is that the code is using temporary AWS credentials to access the unsigned document. This will be replaced with using long-lasting AWS credentials once we have set these up.

### CODE REVIEW

File: src/test/postman/Document_Signing_API.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+							"raw": "{\n    \"document_location\": \"https://document-api-images-cidev.s3.eu-west-2.amazonaws.com/9616659670.pdf\",\n    \"document_type\": \"certified-copy\",\n    \"signature_options\": [\n        \"cover-sheet\"\n    ]\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{base_url}}/document-signing-api/document-signing/sign-pdf",
+							"host": [
```
Comment (hwilliams1) [inline src/test/postman/Document_Signing_API.postman_collection.json:28]: formatting in this file seems to use tabs instead of spaces

**Review by hwilliams1 — CHANGES_REQUESTED** [review]
Couple of comments that may need to be looked at/changed regarding the formatting etc

File: src/main/java/uk/gov/companieshouse/documentsigningapi/dto/SignPdfResponseDTO.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package uk.gov.companieshouse.documentsigningapi.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SignPdfResponseDTO {
+
+    @JsonProperty("signed_property_location")
```
[the remark this answers is not in the document]
Comment (VaughanJackson123) [reply src/main/java/uk/gov/companieshouse/documentsigningapi/dto/SignPdfResponseDTO.java:7]: Good spot! I will change that.

File: src/test/postman/Document_Signing_API.postman_collection.json (hunk 1 above)
Comment (VaughanJackson123) [reply src/test/postman/Document_Signing_API.postman_collection.json:28]: I'll look into this.

**Pull request merged**